openvdb_houdini/SOP_OpenVDB_Merge.cc

```diff
@@ -0,0 +1,296 @@
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "scalarmode"
+            label   "Scalar Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "maximum"       "Maximum"
+                "minimum"       "Minimum"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name    "distancemode"
+            label   "Distance Mode"
+            type    ordinal
+            default { "sdfunion" }
+            menu {
+                "none"          "None"
+                "sdfunion"      "SDF Union"
+                "sdfintersect"  "SDF Intersect"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "vectormode"
+            label   "Vector Mode"
+            type    ordinal
+            default { "add" }
+            menu {
+                "none"          "None"
+                "add"           "Add"
+                "multiply"      "Multiply"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name    "boolmode"
+            label   "Bool Mode"
+            type    ordinal
+            default { "topounion" }
+            menu {
+                "none"          "None"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+            joinnext
+        }
+        parm {
+            name    "pointsmode"
+            label   "Points Mode"
+            type    ordinal
+            default { "pointmerge" }
+            menu {
+                "none"          "None"
+                "pointmerge"    "Merge"
+                "topounion"     "Activity Union"
+                "topointersect" "Activity Intersection"
+            }
+        }
+        parm {
+            name        "useallgridsfortopo"
+            label       "Use all Grids for Activity Modes"
+            type        toggle
+            default     { "0" }
+            disablewhen "{ fogmode != topounion fogmode != topointersect scalarmode != topounion scalarmode != topointersect distancemode != topounion distancemode != topointersect vectormode != topounion vectormode != topointersect boolmode != topounion boolmode != topointersect pointsmode != topounion pointsmode != topointersect }"
```

Comment:
Such complicated disablewhens scare me....  I'm not going to know how to activate this!

It sounds almost, if I guess to what this does, you instead want a default Activity operation to be provided or something?